guidelines/technical-guidelines/08-telepresence-guide.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Using Telepresence for local Kyma development
+---
+
+This document is a general guide to local development with [Telepresence](https://www.telepresence.io/) (tested on version `0.101`).
```

Comment:
(tested on version `0.101`). -> It's a common practice in Kyma docs to avoid putting such information in parenthesis. Please incorporate version information into the sentence or make it a note.